src/modules/shared/components/MediaCard/MediaCard.module.scss

```diff
@@ -160,12 +160,12 @@ $c-media-card-no-video-intended-spacing: 0.9rem;
 
 	:global(.c-card__subtitle-wrapper),
 	:global(.c-card__children-wrapper) {
-		line-height: $line-height-md * 2;
+		line-height: $line-height-md * 2rem;
```

Comment:
This is not equivalent, be careful of unforeseen side-effects
https://codepen.io/ian-emsens-shd/pen/eYaRpvW

Reply:
i know, i changed 2rem => $line-height-md * 2
but that broke something

so with the rem it should be fixed